docs/admin/cluster-large.md

```diff
@@ -105,6 +109,15 @@ To avoid running into cluster addon resource issues, when creating a cluster wit
 
 For directions on how to detect if addon containers are hitting resource limits, see the [Troubleshooting section of Compute Resources](../user-guide/compute-resources.md#troubleshooting).
 
+### Allowing minor node failure at startup
+
+For various reasons (see [#18969](https://github.com/kubernetes/kubernetes/issues/18969) for more details) running
+`kube-up.sh` with a very large `NUM_NODES` may fail due to a very small number of nodes not coming up properly.
+Currently you have two choices: restart the cluster (`kube-down.sh` and then `kube-up.sh` again), or before
+running `kube-up.sh` set the environment variable `ALLOWED_NOTREADY_NODES` to whatever value you feel comfortable
+with. This will allow `kube-up.sh` to succeed with fewer than `NUM_NODES` coming up. Depending on the
+reason for the failure, those additional nodes may join later or the cluster may remain at a size of
+`NUM_NODES - ALLOWED_NOTREADY_NODES`.
```

Comment:
Yeah, I think we should default `EXIT_ON_WEAK_ERROR` to false (except in the flaky suite version of the test). As long as we default ALLOWED_NOTREADY_NODES to 0, then people will get the same behavior as they get today without setting any flags, and if they want to allow minor node failure they only have to change ALLOWED_NOTREADY_NODES.